docs/update-module.md

```diff
@@ -0,0 +1,104 @@
+# Updating a module
+
+When you have improvements or fixes to a module or want to address limitations, you can contribute updates. You fork the module you want to work with.
+
+?> **Tip:** If you're creating a module, see [Contributing to the IBM Cloud Terraform modules project](/contribute-module.md).
+
+## Overall process
+
+1.  Set up your development environment.
+1.  Fork the repo and clone it locally.
+1.  Make your changes.
+1.  Test your changes with example code.
+1.  Open a pull request.
+
+## Before you begin
+
+[inc-prereqs-dev.md](inc-prereqs-dev.md ':include')
+
+## Fork, clone, and initialize the repo
+
+Fork the module repo that you want to update.
+
+1.  Navigate to the repository that you want to update.
+1.  Fork the repo by clicking **Fork** in the top-right corner of the page.
+
+    - Clear the **Copy the main branch only** option to copy all branches to your fork.
```

Comment:
I think this is optional, typically I would only want the main branch not all the forks